pkg/kubelet/kubelet.go

```diff
@@ -2847,7 +2847,7 @@ func (kl *Kubelet) canResizePod(pod *v1.Pod) (bool, v1.PodResizeStatus) {
 			}
 		}
 		if utilfeature.DefaultFeatureGate.Enabled(features.MemoryManager) {
```

Comment:
we can either repurpose this PR to add more cleanup or file another one to remove the remaining feature gate reference

Reply:
I would prefer saving reviews cycles and wrap it up in this PR. Are you ok with that?

Reply:
Shall we remove the feature gate condition completely?

Reply:
handled in https://github.com/kubernetes/kubernetes/pull/128728 , no further changes needed in this PR